storybookjs/storybook
# FIX postcss options missing in default webpack config && UPDATE dependencies

Fixes: #1084

## What I did
CHANGE the webpack config so it has postcss options (the same as create-react-app)

## How to test
- `npm install`
- `npm run bootstrap `
- `cd examples/cra-storybook`
- `npm run start` - should work = `<body>` has grey background
- `npm run storybook` - should work = `<body>` has grey background
